### PR TITLE
Add more remote_file examples

### PR DESCRIPTION
**Pull request by tas50**
Fill it out a bit more

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: data/infra/resources/remote_file.yaml [hunk 1]
```diff
@@ -138,7 +138,7 @@ properties_list:
       A Hash of custom headers. For example:
 
       ```ruby
-      headers({ "Cookie" => "user=grantmc; pass=p@ssw0rd!" })
+      headers({ "Cookie" => "user=some_user; pass=p@ssw0rd!" })
```
Comment (kagarmoe) [inline data/infra/resources/remote_file.yaml]:
```suggestion
      headers({ "Cookie" => "user=user_name; pass=p@ssw0rd!" })
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Tiny change. Could you update the `file` resource next?

**Comment by tas50** [comment]
These are the nast ones I've avoided automating in the past. When I get some time I'll slowly update other ones

**Review by kagarmoe — APPROVED** [review]
tenor-232813978

**Pull request merged**